scripts/webpack/webpack-resources.js

```diff
@@ -154,6 +157,10 @@ module.exports = {
           extensions: ['.ts', '.tsx', '.js']
         },
 
+        resolveLoader: {
+          modules: ['node_modules', path.join(__dirname, '../node_modules'), path.join(__dirname, '../../node_modules')]
```

Comment:
Could this automatically resolve deps for packages that didn't explicitly include them? I'm worried with this change that a package could resolve to a dep of another package but then fail when consumed as an NPM package.

Reply:
All webpack resources / devdeps reside in scripts. There could be phantom webpack loaders that we reference in theory, but that's kind of a problem of having a tooling package inside a yarn workspace monorepo in general. Given that this is a single source of truth (all our webpack configs are in /scripts), we should be okay

Reply:
Ok, I see this is for resolving loaders, not deps, so my concern doesn't apply.